### PR TITLE
fix: Fixed import excel products and patients #194

### PR DESCRIPTION
**Pull request by jmartinacu**
## Description
Fix Import Excel of Patients and Products into db. Due to the new changes in update of base_crud.py.

## Related Issue(s)
- #194 

## Checklist
- [x] I have read and followed the project's contributing guidelines.
- [x] I have tested the changes locally.
- [x] I have made corresponding changes to the documentation if necessary.
- [x] My code follows the project's code style guidelines.

## Reviewers
- @alvgonfri 

## Screenshots 
image

### CODE REVIEW

File: src/modules/cyc/warehouse/service.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+from uuid import uuid4
 from pydantic import UUID4
 
 from fastapi import HTTPException, status
+from pymongo import UpdateMany, UpdateOne, InsertOne
 
 from src.core.deps import DataBaseDep
-from src.core.database.mongo_types import InsertOneResultMongo, DeleteResultMongo, UpdateResult
+from src.core.database.mongo_types import InsertOneResultMongo, DeleteResultMongo, UpdateResult, BulkWriteResult
 from src.modules.cyc.warehouse import model
```
Comment (alblopnov) [inline src/modules/cyc/warehouse/service.py:12]: Please, delete the duplicated method get_products_service from this service

**Review by alblopnov — CHANGES_REQUESTED** [review]
I requested some changes. If you could, please also change the single quoted ['id`] in line 36 of test_shared_user to fix the problem we talked about this morning

**Review by alvgonfri — CHANGES_REQUESTED** [review]
The feature works great, but you should make the changes requested by @alblopnov and also another change at `helpers.py`: that string must be written entirely on a single line.

image

**Review by alvgonfri — APPROVED** [review]
Everything has been corrected.

**Pull request merged**